### PR TITLE
Add automation to import Pontoon TMX

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: .github/scripts/download_tmx.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    for locale in locales:
+        # Create locale folder if missing
+        locale_path = os.path.join(root_path, locale)
+        if not os.path.isdir(locale_path):
+            os.mkdir(locale_path)
+
+        print(f"Downloading TMX for {locale}")
+        try:
+            tmx_filename = f"{locale}.all-projects.tmx"
```
Comment (gueroJeff) [inline .github/scripts/download_tmx.py]: This is just a nit, but could we change the naming convention to {locale}_pontoon.tmx

**Comment by flodolo** [comment]
cc @mathjazz 

@gueroJeff 
I believe this would work, with a few caveats:
* I tried to set it up to create pull requests, and it doesn't work great, because you can't do much besides approving it (the number and size of files is too large). The current version commits directly.
* It's a **lot** of large files.

Let me know if I should merge and try to trigger it manually.

**Review by mathjazz — APPROVED** [review]
Sounds like a plan! Nice work!

 I'll assign the bug to you and attach the PR.

**Comment by flodolo** [comment]
BTW, you can see the output of this in my fork
https://github.com/flodolo/mt-training-data/

The cronjob is set to run on the 1st day of the month, but can be run manually from actions.

**Comment by gueroJeff** [comment]
This looks great to me, and the output looks exactly like what we need for this case (and thanks for the naming convention change). 1st day of the month is great cadence.

Thanks for putting this together so quickly (and on your day off too)!

**Pull request merged**